acl/README.md

```diff
@@ -13,3 +13,47 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+# Testing acl on Linux
+
+## Build, Install and Test the acl core plan
+
+### Build and Install
+
+Open a terminal at the root directory of the core-plans repo.  Build and install the package:
+
+```bash
+hab pkg build acl
+source ./results/last_build.env
```

Comment:
It's probably unnecessary to tell someone to open a terminal.

What is sourcing the last_build.env doing here?

Reply:
Only leaving the instructions to run tests as per comment below.